vent/menu.py

```diff
@@ -114,7 +114,8 @@ class VentApp(npyscreen.NPSAppManaged):
 
     def onStart(self):
         """ Override onStart method for npyscreen """
-        self.addForm("MAIN", VentForm, name="Vent\t\t\t\t\t\t\t\tPress ^T to toggle help", color="IMPORTANT")
+        version = pkg_resources.require("vent")[0].version
```

Comment:
I didn't realize we had a version function already. I believe that function is outdated as it looks for a VERSION file which we no longer have. I think I'll go ahead and update the version function, then import and use that accordingly.